build/build.go

```diff
@@ -26,3 +26,13 @@ func Runfile() string {
 	// return converted output value
 	return output
 }
+
+// Native function to call Linux command
+func Native() string {
+	cmd, err := exec.Command("/bin/sh", "/Users/richard/Desktop/Go/jekyll-server-build/build/echo.sh").Output()
+	if err != nil {
+		fmt.Printf("error %s", err)
+	}
+	output := string([]byte(cmd))
```

Comment:
Since `output` is already a `[]byte`, we can skip this extra conversion and we can reduce this to:

```suggestion
	return string(output)
```

Reply:
Noted and updated